.phan/config.base.php

```diff
@@ -34,6 +34,7 @@
  *      - wp-cli: Stubs from php-stubs/wp-cli-stubs.
  *      - wpcom: Stubs from .phan/stubs/wpcom-stubs.php.
  *   - +stubs: (array) Like 'stubs', but setting this does not clear the defaults.
+ *   - internal_stubs: (array) Stubs provided by Phan to use with various PHP extensions like xdebug, zip, etc. See https://github.com/phan/phan/wiki/How-To-Use-Stubs#internal-stubs.
```

Comment:
I've been trying to keep these in alphabetical order.

Also, it might make more sense to name this something like "php_extensions_needed"?

Reply:
Yeah, good call, and I haven't noticed the alphabetical order before, fixed.